.github/workflows/bump-image-tag.yaml

```diff
@@ -27,7 +27,7 @@ jobs:
       uses: fjogeleit/yaml-update-action@main
       with:
         valueFile: prebuilt-config/bitops-tag.yaml
-        propertyPath: 'tags.bitops_base'
+        propertyPath: 'bitops_base_tag'
         value: ${{ env.BASE_TAG}}
         commitChange: true
         updateFile: true
```

Comment:
@PhillypHenning I'm afraid the multi-step logic updating the bitops tag via commit is out of date since we agreed on introducing the `dev` tag that's generated from the `main` branch.

The problem is that TAG file update (including for releases) happens in the `main` branch too.
Instead of that we could be just taking the tag from the original release version and consider that as a source of truth for tagging the "stable" BitOps docker images via GH Workflows in one go without multi-step tag file update.

Are you fine with this approach? Do you see any corner cases going that way?
I can work on refactoring the workflow in another PR if agreed.
WDYT?

Reply:
> we could be just taking the tag from the original release version
@armab Is this to say that we use a GitHub tag based workflow, and whatever the GitHub Tag is is what we use for the image versions?

@PhillypHenning remind me why we had the multi-step approach (i.e. update a file)?

Reply:
@mickmcgrath13 
The idea behind it was that if anything source wise changed it would regenerate the base tag, which then would bump the tag file, which was the trigger for the loaded images. 

@armab 
I like the approach you're suggesting 👍